test/example.test.js

```diff
@@ -1,16 +1,16 @@
 // IMPORT MODULES under test here:
 // import example from '../src/example.js';
 
-const test = QUnit.test;
+// const test = QUnit.test;
 
-test('time to test a function', function(assert) {
-    //Arrange
-    // Set up your parameters and expectations
+// test('time to test a function', function(assert) {
+//     //Arrange
+//     // Set up your parameters and expectations
 
-    //Act 
-    // Call the function you're testing and set the result to a const
+//     //Act 
+//     // Call the function you're testing and set the result to a const
 
-    //Assert
-    // Make assertions about what is expected valid result
-    assert.equal(true, false);
```

Comment:
This is the problem with Travis. If you set both of the values to the same boolean (true, true  OR false, false) your test should pass and Travis should be happy :)